src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -628,11 +632,10 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.WAVPACK;
 		} else if (value.contains("mlp")) {
 			format = FormatConfiguration.MLP;
+		} else if (value.equals("openmg")) {
+			format = FormatConfiguration.ATRAC;
 		} else if (value.contains("atrac")) {
 			format = FormatConfiguration.ATRAC;
-			if (media.getCodecV() == null && !media.getContainer().equals(FormatConfiguration.RM)) {
-				media.setContainer(FormatConfiguration.ATRAC);
-			}
 		} else if (value.equals("jpeg")) {
```

Comment:
@Sami32 I think you misunderstood me. I'm not saying we shouldn't change it, I'm just saying that we must make sure that changing it doesn't break anything.

I'm not really sure if ATRAC is only an audio codec or also a file format, but if it's also a file format how will the correct "container" be set without this code?